Makefile.am

```diff
@@ -1,8 +1,24 @@
+# Work around inconsistency in AX_CODE_COVERAGE defining
+# AM_DISTCHECK_CONFIGURE_FLAGS only in some branches
+if !CODE_COVERAGE_ENABLED
+AM_DISTCHECK_CONFIGURE_FLAGS=
```

Comment:
For consistency with other parts of Makefile.am, maybe put spaces around the equal sign in assignments here (and elsewhere).